### ISSUE TITLE
Flatpak Bleachbit Won't Launch After Update "Traceback (most recent call last):"

### ISSUE THREAD

**Issue by KrauzerPH94** [bleachbit/bleachbit#1566]
## Bug Description
App won't launch after last update, no error message, it just hangs without response.
I'm currently on **version v4.5.0**, app installed via [Flatpak using the Flathub](https://flathub.org/apps/org.bleachbit.BleachBit) repository.

## How to Reproduce
There are two ways to reproduce

### Via GUI
1. Install the app using your desired software center;
2. Try to launch it via your app launcher and/or shortcut;
3. Observe the app process hanging then closing itself.

### Via CLI
1. Install and run the application:
```bash
$ flatpak install org.bleachbit.BleachBit
$ flatpak run org.bleachbit.BleachBit
```
2. Observe the feedback error:
```bash
Traceback (most recent call last):
  File "/app/bin/bleachbit", line 28, in <module>
    from bleachbit import _
ModuleNotFoundError: No module named 'bleachbit'
```

- **Bug Ocurrence:** everytime I open the app, using either one of the above methods
- **Expected behavior:** for the app to open, either via GUI or CLI, and display me it's window

## My Environment
- OS: Fedora 38
- DE: GNOME 44.5
- Kernel: 6.5.7
- WM: Xorg (X11)
- App version: v4.5.0
- App repo: [Flathub](https://flathub.org/)

**Comment by az0** [comment]
How about <a href="https://www.bleachbit.org/news/bleachbit-451-beta"> version 4.5.1</a> ?

**Comment by KrauzerPH94** [comment]
[The mentioned version (4.5.1)](https://www.bleachbit.org/news/bleachbit-451-beta) is still not available in the [Flathub repository](https://flathub.org/apps/org.bleachbit.BleachBit), I prefer using this app as a flatpak.

As far as I understand, the link you sent is related to a native version of Bleachbit, or am I missing something?

**Comment by az0** [comment]
That's all I got, so I'm closing the ticket

**Comment by az0** [comment]
> Do you prefer flathub to track Tags or Releases? 

TBH, I have mixed feelings about FlatHub

>I noticed that you haven't "released" 4.6.0 while it's tagged.

The release process has many steps, and this is on the to-do list.

> On the other hand, you tagged the broken 4.5.0 (no naming difference between beta and release versions) but it was marked as pre-release.

For any version x.y.z where y or z is odd, then it's not a final release.
